### PR TITLE
Use CDN if in production, fixes #15

### LINKED ISSUE #15: Use CDN if in production
Related files: https://github.com/ayan-b/blog/blob/master/_includes/header.html 
https://github.com/ayan-b/blog/blob/master/_includes/footer.html

### PR DESCRIPTION
**Pull request by xevenheaven**
Added the CDN links for:
* https://cdnjs.cloudflare.com/ajax/libs/jquery/3.2.1/jquery.min.js
* https://cdnjs.cloudflare.com/ajax/libs/materialize/1.0.0/js/materialize.min.js

Please take a look!

### CODE REVIEW

File: _includes/header.html [hunk 1]
```diff
@@ -8,7 +8,12 @@
       <!--Import Google Icon Font-->
       <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
       <!--Import materialize.css-->
-      <link type="text/css" rel="stylesheet" href="{{site.baseurl}}/assets/materialize/css/materialize.min.css"  media="screen,projection"/>
+      {% if jekyll.environment == "development" %}
+      <link href="{{site.baseurl}}/assets/materialize/css/materialize.min.css" rel="stylesheet" media="screen,projection">
+      {% endif %}
+      {% if jekyll.environment == "production" %}
+      <link href="https://cdnjs.cloudflare.com/ajax/libs/materialize/1.0.0/css/materialize.min.css" rel="stylesheet" media="screen,projection">
+      {% endif %}
```
Comment (ayan-b) [inline _includes/header.html]: Please add `text/css`. If you have any specific reason for not adding let me know. :smile:

File: _includes/header.html [hunk 2]
```diff
@@ -8,7 +8,12 @@
       <!--Import Google Icon Font-->
       <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
       <!--Import materialize.css-->
-      <link type="text/css" rel="stylesheet" href="{{site.baseurl}}/assets/materialize/css/materialize.min.css"  media="screen,projection"/>
+      {% if jekyll.environment == "development" %}
```
Comment (ayan-b) [inline _includes/header.html]: Hmm, are you sure this is the correct way to do this? :thinking: See deploy preview at netlify, it does not load the CDN.

**Comment by ayan-b** [comment]
Also, don't forget to add your name to the Contributors section! :wink:

File: _includes/header.html (hunk 2 above)
Comment (xevenheaven) [reply _includes/header.html]: We may need to change the build command for production, but I don't see it anywhere in the code base where I can update it.

Here's the Jekyll docs for determining the environment:
https://jekyllrb.com/docs/configuration/environments/

It mentions to export the environment variable:
```bash
JEKYLL_ENV=production jekyll build
```

Anything I can do to update the build command?

File: _includes/header.html (hunk 1 above)
Comment (ayan-b) [reply _includes/header.html]: Awesome. :wink:

File: _includes/header.html (hunk 2 above)
Comment (ayan-b) [reply _includes/header.html]: Hmm, I guess I need to change the build command at netlify. I will merge it.

Comment (xevenheaven) [reply _includes/header.html]: Checked the site after merge, it's calling the CDN links. 😄 
And yup, I think updating the netlify build command should make it work for deploy preview.

Thanks for merging @ayan-b!

**Comment by ayan-b** [comment]
Here https://ayan-b.github.io/blog/ your changes are working. :+1:

**Pull request merged**